impeller/typographer/backends/stb/typeface_stb.h

```diff
@@ -6,9 +6,9 @@
 
 #include "flutter/fml/macros.h"
 #include "flutter/fml/mapping.h"
+#include "flutter/third_party/stb/stb_truetype.h"
```

Comment:
I think that leaving the include path as is should hopefully still work. I'm a bit worried that the include rewriter that imports the source into the internal repo might be confused by this change.